sfdoc/publish/views.py

```diff
@@ -138,7 +140,16 @@ def review(request, pk):
         return HttpResponseRedirect('../')
     else:
         form = PublishToProductionForm()
+    try:
+        base_url = get_community_base_url()
+    except Exception as error:
+        print(error)
+        print('Error getting base URL')
```

Comment:
Can you explain the error handling here? I think this is going to print to somewhere we'll never see, and then break with a NameError on line 149. Would be easier to track down if you don't swallow the exception.

Reply:
@davisagli this wasn't ready for primetime